packages/mui-component-mapper/src/files/field-array.js

```diff
@@ -0,0 +1,248 @@
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { useFormApi, FieldArray } from '@data-driven-forms/react-form-renderer';
+
+import Grid from '@material-ui/core/Grid';
```

Comment:
So here, we actually want to use shorthand imports `import {Grid, Button, ...} from 'material-ui/core'`. The babel plugin is taking care of importing correct components for the cjs/esm builds.